index.js

```diff
@@ -106,7 +115,8 @@ function isReactFragment(openingElement) {
   )
 }
 
-function applyAttributes(t, openingElement, componentName, sourceFileName, componentAttributeName, elementAttributeName, sourceFileAttributeName) {
+function applyAttributes(t, openingElement, componentName, sourceFileName, attributeNames, ignoreComponentsFromOption) {
+  const [ componentAttributeName, elementAttributeName, sourceFileAttributeName ] = attributeNames;
```

Comment:
@JoshMiers-FS if it wasn't clear, the `...` above is replaced by this line here. They both have the effect of extracting each of the values of the array into their own variables. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax for more info about `...` and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment for more about this replacement.